gossip/src/cluster_info.rs

```diff
@@ -2032,26 +2020,14 @@ impl ClusterInfo {
         let pings = pings
             .into_iter()
             .map(|(addr, ping)| (addr, Protocol::PingMessage(ping)));
-        send_gossip_packets(pings, recycler, response_sender, &self.stats);
-        self.handle_batch_ping_messages(ping_messages, recycler, response_sender);
+        send_gossip_packets(pings, response_sender, &self.stats)?;
```

Comment:
Same here, let us not change error handling patterns in gossip in this PR to keep the scope narrow.

Reply:
The current code on main ends up using `Packet::populate_packet`, which also returns a `Result`, but it is logged  in this function:

https://github.com/anza-xyz/agave/blob/ddefc131edf65b3ca5dfcdfc2af02e420834b5db/perf/src/packet.rs#L684

I guess I could write a similar method for `BytesPacketBatch` and stop sprinkling the error in gossip.

I didn't do that before, because I felt like ignoring a serialization error is a smell, but I agree with you that it adds unnecessary diff and that we should propose error handling changes as separate PRs. So I will just remove the error handling changes in the next push.